pom.xml

```diff
@@ -96,37 +91,116 @@
         <module>activation-registry</module>
     </modules>
 
-    <profiles>
-        <!--
-            This profile contains modules that should only be built
-            but not installed or deployed.
-        -->
-        <profile>
-            <id>build-only</id>
-            <modules>
-                <module>demo</module>
-            </modules>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-            </activation>
-        </profile>
-        <!--
-            This profile is used for deploying a Jakarta Activation
-            final release.
-
-            Activating this profile manually for deployment causes
-            the above profile to be deactivated, which works around
-            an apparent bug in maven that prevents me from manually
-            deactivating a profile.  This profile purposely has none
-            of the modules I don't want to be deployed.
-        -->
-        <profile>
-            <id>oss-release</id>
-        </profile>
-    </profiles>
+    <dependencyManagement>
+        <dependencies>
+            <!--Our artifacts-->
```

Comment:
drop this comment, it is not our artifact

Reply:
Removed.